infra/nnfw/cmake/options/options_armv7l-tizen.cmake

```diff
@@ -11,6 +11,5 @@ option(GENERATE_RUNTIME_NNAPI_TESTS "Generate NNAPI operation gtest" OFF)
 option(ENVVAR_ONERT_CONFIG "Use environment variable for onert configuration" OFF)
 
 option(DOWNLOAD_OPENCL_HEADERS "Download Opencl_headers source" ON)
-option(DOWNLOAD_TENSORFLOW_GPU "Download Tensorflow GPU delegate source" ON)
-option(BUILD_GPU_CL "Build gpu_cl backend" ON)
+option(BUILD_GPU_CL "Build gpu_cl backend" OFF)
```

Comment:
@YiHyunJin Why you turn off build from tizen platform ?

Reply:
I turned it off to confirm. If you compress and upload the TF2.7 code now, it will not upload if the size is over 50MB.
So, I am trying to link tf2.7 directly through nnfw.spec.

Reply:
I turned it off to confirm. I compress and try to upload the TF2.7 code. But it did not upload. because the size is over 50MB.
So, I am trying to link tf2.7 directly through nsfw.spec.

Reply:
What do you mean by "link tf2.7 directly through nnfw.spec" ?

Reply:
I think.. we need tf-lite 2.7 source code or lib.
I try to upload tf-lite 2.7 souce code, but It was failed..
So I will try to link "tensorflow-lite-devel" version 2.7
Is this solution incorrect??...

Reply:
Have you check https://review.tizen.org/gerrit/#/admin/projects/platform/upstream/tensorflow2 is able to provide such package ? IMHO, I do not think so. Or you should split compression.

Reply:
`Or you should split compression.`
oh it is good idea. : )
Thanks. I will try to split compression

Reply:
@YiHyunJin Yes.